src/index.ts

```diff
@@ -4,23 +4,10 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-
-export { getBearer } from "./download/bearerToken";
-export { RestApi } from "./download/exchangeTypes";
-export { ramlToolLogger } from "./common/logger";
-
-import * as amf from "amf-client-js";
-export { amf };
-export { model } from "amf-client-js";
-
-export * from "./generate";
-
+export * as amf from "amf-client-js";
+export * as amfParser from "./common/parser";
```

Comment:
parser is too generic which is why I made little specific. I am not sure if we want to enforce the rule that export name must always match the directory name, specially for the features in common. For the command functionality it make sense because they are already grouped

Reply:
Just to take this to the logical conclusion, to satisfy both having the export name match the directory name AND keep the export name more specific, should the directory be changed to amfParser? I'm not a huge fan of this, but it would satisfy what both of you are saying.

Reply:
make sense to me. I will check with the team and open another PR
FYI - parser is file not directory